CentralManagement/ChocoCCM Command Reference/Remove-CCMStaleDeployment.md

```diff
@@ -0,0 +1,86 @@
+---
+external help file: ChocoCCM-help.xml
+Module Name: ChocoCCM
+online version:
+schema: 2.0.0
+---
+
+# Remove-CCMStaleDeployment
+
+## SYNOPSIS
+Removes stale CCM Deployments
+
+## SYNTAX
+
+```
+Remove-CCMStaleDeployment [-Age] <String> [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Remove stale deployments from CCM based on their age and run status.
+
+## EXAMPLES
+
+### EXAMPLE 1
+```
+Remove-StaleCCMDeployment -Age 30
+```
+
+## PARAMETERS
+
+### -Age
+The age in days to prune
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Position: 1
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```

```

Comment:
I'd suggest this have a default value (30 or 60 days perhaps) and folks can be more specific if they prefer different time values.

I'd also suggest allowing folks to provide a direct DateTime value and pruning all deployments older than that (calculate # of days or whatever if the endpoint we're using for this demands it).